src/dash/dashgenerator.py

```diff
@@ -0,0 +1,50 @@
+# /********************************************************************************
+# * Copyright (c) 2023 Contributors to the Eclipse Foundation
+# *
+# * See the NOTICE file(s) distributed with this work for additional
+# * information regarding copyright ownership.
+# *
+# * This program and the accompanying materials are made available under the
+# * terms of the Apache License 2.0 which is available at
+# * http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * SPDX-License-Identifier: Apache-2.0
+# ********************************************************************************/
+
+"""Methods to generate a notice file from given dependencies."""
+
+from licensevalidator.lib.dependency import DependencyInfo
+
+def generate_clearlydefined_input_file(
```

Comment:
`Clearlydefined` is the specification for a single line of this output file. The tool that accepts this multiline file of strings in clearlydefined format is Eclipse dash, also the file itself is called `dashgenerator`. I would recommend we also call this method `generate_dash_input_file`.

Reply:
@doosuu , it is initial implementation of input file, that is ClearlyDefined input. Next stage will be extend it with dash tool usage that will generate DEPENDENCIES file, that will be only dash specific format.

Dash under hood performs verification in front of 2 datasource - 
1) ClearlyDefined one - https://clearlydefined.io/
2) Eclipse Specific one (defined curations) - https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/tree/master

Dash License Tool itself will only care about existence of Issues, that lead to curation creation in case if component "false-positive" due to verification in front of ClearlyDefined datasource. 

Therefore I would like to keep this name for function, to be able distinguish real Dash Tool implementation later